DXMainClient/Program.cs

```diff
@@ -89,13 +79,13 @@ static void Main(string[] args)
 
             for (int arg = 0; arg < args.Length; arg++)
             {
-                string argument = args[arg].ToUpper();
+                string argument = args[arg].ToUpperInvariant();
 
                 switch (argument)
                 {
                     case "-NOAUDIO":
                         // TODO fix
-                        throw new NotImplementedException("-NOAUDIO is currently not implemented, please run the client without it.".L10N("UI:Main:NoAudio"));
+                        throw new NotImplementedException("-NOAUDIO is currently not implemented, please run the client without it.");
```

Comment:
Why remove this localization string that will show up in the message box to the user later?